bin/upgrade.bash

```diff
@@ -52,6 +52,18 @@ prepare() {
 
   snippets_check prepare "${snippets}"
 
+  # Create a configmap. This fails if done twice.
+  if [[ "${CK8S_CLUSTER:-}" =~ ^(sc|both)$ ]]; then
+    if ! record_migration_prepare_begin sc; then
+      log_fatal "prepare already started in sc (kubectl delete -n kube-system configmap apps-upgrade to try again)"
+    fi
+  fi
+  if [[ "${CK8S_CLUSTER:-}" =~ ^(wc|both)$ ]]; then
+    if ! record_migration_prepare_begin wc; then
+      log_fatal "prepare already started in wc (kubectl delete -n kube-system configmap apps-upgrade to try again)"
+    fi
```

Comment:
suggestion: Move the `log_fatal` inside the function instead of assuming what a non-zero exit code means.